lib/features/mailbox_dashboard/presentation/mailbox_dashboard_controller.dart

```diff
@@ -1,23 +1,27 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 import 'package:get/get.dart';
+import 'package:jmap_dart_client/jmap/account_id.dart';
 import 'package:jmap_dart_client/jmap/core/session/session.dart';
 import 'package:model/model.dart';
 import 'package:tmail_ui_user/features/base/base_controller.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/constants/mailbox_dashboard_constants.dart';
 
 class MailboxDashBoardController extends BaseController {
 
   final scaffoldKey = GlobalKey<ScaffoldState>();
-  final mailboxCurrent = PresentationMailbox.createMailboxEmpty().obs;
-  final mailCurrent = PresentationThread.createThreadEmpty().obs;
+  final selectedMailbox = PresentationMailbox.presentationMailboxEmpty.obs;
+  final selectedEmail = PresentationEmail.presentationEmailEmpty.obs;
+  final accountId = Rxn<AccountId>();
```

Comment:
Why not `AccountId?` ? You created a default empty ID, but, IMO, it is not to make sense.
For the null value, you can prevent to make the request instead of make a bad request with empty account ID, then handling the error.